cmd/develop/main.go

```diff
@@ -26,6 +26,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	cmd.SilenceUsage = true
```

Comment:
Nothing to do with the fix, and only affects developers rather than main CLI users, but cobra's default behaviour of printing usage help when a command fails is annoying 😅